src/Annotation/ApiFilter.php

```diff
@@ -58,7 +58,7 @@ public function __construct($options = [])
         }
 
         if (!is_a($options['value'], FilterInterface::class, true)) {
-            throw new InvalidArgumentException(sprintf('The filter class "%s" does not implement "%s".', $options['value'], FilterInterface::class));
+            throw new InvalidArgumentException(sprintf('The filter class "%s" does not implement "%s". Did you forget a use statement ?', $options['value'], FilterInterface::class));
```

Comment:
isn't there a test to update on this?

Reply:
I think you’re right ! I update test too